lib/email_center_api/version.rb

```diff
@@ -1,3 +1,3 @@
 module EmailCenterApi
-  VERSION = "0.0.3"
+  VERSION = "0.0.4"
```

Comment:
This has been stable in production for 3+ years. Go on, do a version 1 :P

Reply:
Will wait one more year, until we release some major update ;)

Reply:
😂